src/graphics/Screen.cpp

```diff
@@ -305,12 +305,25 @@ static bool haveGlyphs(const char *str)
 /// Used on eink displays while in deep sleep
 static void drawDeepSleepScreen(OLEDDisplay *display, OLEDDisplayUiState *state, int16_t x, int16_t y)
 {
+    const char *idText = owner.short_name;
+
```

Comment:
Don't worry too much about creating `idText` for common style. The only reason I used it in `drawScreensaverOverlay` was because it helped me read the code, so only use it here if it helps you.